### DISCUSSION TITLE
How do I import a custom SVG image?

### DISCUSSION THREAD

**Discussion by ElHex** [quasarframework/quasar#10624] [General - Components / Directives / etc]
Hello! I'm a beginner starting with Quasar, I'm trying to import a TipTap Editor from a old project of mine, currently trying to add the toolbar which has this requirement:

`import remixiconUrl from '/public/assets/remixicon.symbol.svg'`

this sadly gives me this error 

`Cannot find module './public/assets/remixicon.symbol.svg' or its corresponding type declarations.Vetur(2307)`

I'm trying to load it like this:

```
<template>
  <button
    class="menu-item"
    :class="{ 'is-active': isActive ? isActive(): null }"
    @click="action"
    :title="title"
  >
    <svg class="remix">
      <use :xlink:href="`${remixiconUrl}#ri-${icon}`" />
    </svg>
  </button>
</template>
<script lang="ts">
      import remixiconUrl from './public/assets/remixicon.symbol.svg'

export default {
  name: 'MenuItem',
  props: {
    icon: {
      type: String,
      required: true
    },

    title: {
      type: String,
      required: true
    },

    action: {
      type: Function,
      required: true
    },

    isActive: {
      type: Function,
      default: null
    }
  }
}
</script>
```

I'm following both TipTap and Quasar docs and I haven't been able to find a fix for this... any ideas?

**Comment by hawkeye64** [comment]
@ElHex Verify you have an `assets` folder in your `public` folder.

**Comment by ontwikkelfabriek** [comment]
@ElHex 

Welcome to Quasar! For future questions about  how to use Quasar (non bugs) please post on the `discussions` section instead.

https://github.com/quasarframework/quasar/discussions

**Comment by hawkeye64** [comment]
Moving this issue to discussions...
